setup.py

```diff
@@ -61,7 +61,6 @@
     'numpy>=1.9.2',
     'pandas>=0.16.1',
     'scipy>=0.15.1',
-    'bottleneck>=1.0.0'
```

Comment:
From `try:except` block in the `utils.py` file it looks like the use of the bottleneck library is intended to be optional (for those who need the bone rattlin' speed of optimized C ).  Removing bottleneck from the hard requirements is merely a nod to the logic in the `utils.py` file. 

It seems like we might want to either: 

1. remove bottleneck from the requirements (as I have done), or 
2. simplify the utils file to always use the bottleneck library. Either seem logical to me. 

@twiecki 's choice 😃 

(if there is some other reason to keep it as it originally was, I'm happy to revert to that as well... perhaps some sort of thing where `bottleneck` is only supported in certain architectures? )